### DISCUSSION TITLE
Can we create variables greater than 2 dimensions in CVXPY?

### DISCUSSION THREAD

**Discussion by Albert88L** [cvxpy/cvxpy#1973] [Q&A]
Hi, All, here, I have a question. Can we create variables greater than 2 dimensions in CVXPY, such as 4 dimensions (3,3,3,3)? Yalmip can create higher dimensions greater than 2 dimensions, I want to know if CVXPY can do that. Thanks a lot

**Comment by PTNobel** [comment]
Sadly this has been a long-standing goal; see #198
